packages/effect/test/Either.test.ts

```diff
@@ -22,7 +22,7 @@ describe("Either", () => {
       const y = yield* Either.right(2)
       return x + y
     })
-    // eslint-disable-next-line require-yield
+
```

Comment:
If you can remove the unrelated changes :)

Reply:
Sorry `pnpm lint-fix` added automatically. Reverted back.